lib/src/sidenav/types.ts

```diff
@@ -1,55 +1,78 @@
-type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
+type SVG = React.SVGProps<SVGSVGElement>;
```

Comment:
Why are we only accepting SVG types? Shouldn't we allow them to also pass URLs (`icon: SVG | string`)?